content-planner-motia/steps/scrape.step.py

```
@@ -0,0 +1,81 @@
+            'data': {
+                'requestId': request_id,
+                'url': str(url),
+                'title': title,
+                'content': content,
+                'timestamp': timestamp
+            }
+        })
+        
+    except requests.exceptions.RequestException as e:
+        logger.error(f"HTTP request failed: {str(e)}")
+        raise Exception(f"Firecrawl scraping failed: {str(e)}")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)